libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/InstructionView.java

```diff
@@ -195,16 +184,11 @@ public void subscribe(LifecycleOwner owner, NavigationViewModel navigationViewMo
     lifecycleOwner.getLifecycle().addObserver(this);
     this.navigationViewModel = navigationViewModel;
 
-    navigationViewModel.retrieveRouteProgress().observe(lifecycleOwner, routeProgress -> {
-      if (routeProgress != null) {
-        updateDistanceWith(routeProgress);
```

Comment:
this **CAN NOT** be removed. The InsturctionView has two updates:
* `routeProgress` is for the distance text and the InstructionList update
* `bannerInstructions` below is for maneuver icon and the top banner instruction update

Reply:
Good catch. Missed it. Fixed now